projects/aca-content/folder-rules/src/rule-details/conditions/rule-simple-condition.ui-component.spec.ts

```diff
@@ -90,20 +79,21 @@ describe('RuleSimpleConditionUiComponent', () => {
     fixture = TestBed.createComponent(RuleSimpleConditionUiComponent);
     categoryService = TestBed.inject(CategoryService);
     loader = TestbedHarnessEnvironment.loader(fixture);
+    unitTestingUtils = new UnitTestingUtils(fixture.debugElement, loader);
   });
 
   it('should default the field to the name, the comparator to equals and the value empty', () => {
     fixture.detectChanges();
 
-    expect(getByDataAutomationId(fieldSelectAutomationId).componentInstance.value).toBe('cm:name');
-    expect(getByDataAutomationId('comparator-select').componentInstance.value).toBe('equals');
-    expect(getByDataAutomationId('value-input').nativeElement.value).toBe('');
+    expect(unitTestingUtils.getByDataAutomationId(fieldSelectAutomationId).componentInstance.value).toBe('cm:name');
+    expect(unitTestingUtils.getByDataAutomationId('comparator-select').componentInstance.value).toBe('equals');
+    expect(unitTestingUtils.getByDataAutomationId('value-input').nativeElement.value).toBe('');
```

Comment:
this automation id is duplicated too - please move it to wrapper function or use const to store it